contracts/FYToken.sol

```diff
@@ -23,13 +23,16 @@ contract FYToken is IFYToken, IERC3156FlashLender, AccessControl(), ERC20Permit,
     using CastU256U32 for uint256;
 
     event Point(bytes32 indexed param, address value);
+    event FlashFeeFactorSet(uint256 indexed fee);
     event SeriesMatured(uint256 chiAtMaturity);
     event Redeemed(address indexed from, address indexed to, uint256 amount, uint256 redeemed);
 
     uint256 constant CHI_NOT_SET = type(uint256).max;
 
     uint256 constant internal MAX_TIME_TO_MATURITY = 126144000; // seconds in four years
     bytes32 constant internal FLASH_LOAN_RETURN = keccak256("ERC3156FlashBorrower.onFlashLoan");
+    uint256 constant FLASH_LOANS_DISABLED = type(uint256).max;
```

Comment:
Is the reason that this disables the flash loan because it would overflow?  It took me a second to realize that,  maybe we can comment that?

Reply:
Commented.